infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/TwoPrimariesClusterManager.java

```diff
@@ -0,0 +1,10 @@
+package org.corfudb.infrastructure.logreplication.infrastructure.plugins;
+
+
+public class TwoPrimariesClusterManager extends FileReaderClusterManager {
```

Comment:
Can we change primary -> active (to be consistent) and also add descriptions to these classes.

Reply:
ok